README.md

```diff
@@ -26,3 +26,11 @@ apm install atom-beautify
 ```
 
 Or Settings/Preferences ➔ Packages ➔ Search for `atom-beautify`
+
+## Contribute
+1. Clone the repository
+1. `cd 'atom-beautify'`
+1. Run `npm install`
+1. Run `apm link` (if necessary)
+1. Install [atom-typescript](https://atom.io/packages/atom-typescript) (if using Atom)
+1. Typescript files in `/src` will be compiled to Javascript and saved in `/lib` whenever a file is saved
```

Comment:
The steps are all `1.`.

Reply:
The numbers for numbered lists don't matter in the raw file, it outputs as 1-6:  https://github.com/szeck87/atom-beautify/tree/stevenzunibeautify#contribute

Reply:
Yes, but what about uses -- like myself -- who use text editors such as [Vim](http://www.vim.org/) to view these README.md files without Markdown previews 😝 . The textual file should mean something, and then the Markdown rendering is a luxurious way to view it.

Reply:
Yea, but if you have to add a step in the middle, it's a pain to update.  I updated it here though.

Reply:
> it's a pain to update.

Good. It will encourage us to keep the list of steps short 😜 .

Thanks.